README.md

```diff
@@ -127,6 +127,45 @@ $ pytest -v
 
 Done! You are ready to develop for the RMM OSS project.
 
+### Caching third-party dependencies
+
+RMM uses [CPM.cmake](https://github.com/TheLartians/CPM.cmake) to
+handle third-party dependencies like spdlog, Thrust, GoogleTest,
+GoogleBenchmark. In general you won't have to worry about it. If cmake
+finds an appropriate version on your system, it'll use that (you can
+help that along by setting `CMAKE_PREFIX_PATH` to point to the
+installed location. Otherwise those dependencies will be downloaded as
+part of the build.
+
+If you frequently start new builds from scratch, you might want to
+consider setting the environment variable `CPM_SOURCE_CACHE` to an
+external download directory, which will avoid repeated downloads of
+the thirdparty dependencies.
```

Comment:
```suggestion
If you frequently start new builds from scratch, consider setting the
environment variable `CPM_SOURCE_CACHE` to an external download
directory to avoid repeated downloads of the third-party dependencies.
```